install.sh

```diff
@@ -709,7 +723,7 @@ install_command_line_tool() {
 ############################################################
 setup_automatic_update_checks() {
     if CURRENT_SETTING=$(git config --get githooks.autoupdate.enabled); then
-        if [ "$CURRENT_SETTING" = "Y" ]; then
+        if [ "$CURRENT_SETTING" = "true" ]; then
```

Comment:
This test for example would probably not consider auto updates to be enabled after this change, since it reads the config and will ignore the previous `Y`.
Again, I think reverting the test changes should surface these nicely to fix

Reply:
ah correct, there are some errors from my side!